core/input/vkeys.js

```diff
@@ -12,7 +12,7 @@
 export default {
     0x08: 'Backspace',
     0x09: 'Tab',
-    0x0a: 'NumpadClear',
+    0x0c: 'NumpadClear',
```

Comment:
The physical key is still `Numpad5`, so add that and add a comment why there is a special one here.